Documentation/guides/WindowsOnArm64.md

```diff
@@ -39,8 +39,7 @@ This downloads the latest files in `C:\Program Files\dotnet\sdk-manifests`.
 
 The Mono workloads from dotnet/runtime need a small fix.
 
-Open `C:\Program Files\dotnet\sdk-manifests\7.0.100\microsoft.net.workload.mono.toolchain\WorkloadManifest.json`
```

Comment:
Should we just update the path here to mention `microsoft.net.workload.mono.toolchain.net6` and `microsoft.net.workload.mono.toolchain.net7` now?